pkg/util/ip_reservation_test.go

```diff
@@ -116,8 +130,42 @@ func TestGetUnReservedIPRange(t *testing.T) {
 			errorExpected: false,
 		},
 		{
-			name:            "1 Pending 0 Used",
+			name:            "0 Pending, 1 /24 Used enterprise",
+			cidr:            "192.168.92.0/22",
+			ipRangeSize:     IpRangeSizeEnterprise,
+			pendingIPRanges: make(map[string]bool),
+			cloudProviderReservedIPRanges: map[string]bool{
+				ipsLarge[0]: true,
```

Comment:
I think we can directly use the string values here. makes the test case more readable instead of looking to find the value for the key. Thoughts?

Reply:
/27 has 4 /29 ranges, and ips[i] is in the ith range, so it's a lot easier to think about once understand that. the same goes with ipsLarge. so I think it'd be good to keep them as is. Maybe add some comments to explain that part.

Reply:
Ack,. add more comments